tailwind.config.js

```diff
@@ -43,24 +43,24 @@ module.exports = {
         'pulse-horizontal': 'pulse-horizontal 2s ease-in-out infinite',
       },
       colors: {
-        'blue-900': '#0d0030',
-        'blue-800': '#1a0259',
-        'blue-700': '#3a00d6',
-        'blue-600': '#4b61d5',
-        'blue-500': '#15a1fc',
-        'blue-400': '#bff5fa',
+        accent: '#15a1fc',
+        'accent-dark': '#0d0030',
+        'accent-light': '#bff5fa',
+        'accent-dimmed': '#3a00d6',
+        'accent-semi-dimmed': '#4b61d5',
+        'accent-background': '#1a0259',
 
-        orange: '#f1962e',
-        yellow: '#fed823',
-        pink: '#ff7bed',
-        green: '#01feb6',
+        primary: '#fed823',
+        secondary: '#ff7bed',
+        tertiary: '#01feb6',
         white: '#efecd6',
+        extra: '#f1962e',
 
-        pink50: 'rgba(255, 123, 237, 0.5)',
-        yellow50: 'rgba(254, 216, 35, 0.4)',
-        blue50: 'rgba(21, 161, 252, 0.5)',
-        white50: 'rgba(239, 236, 214, 0.4)',
-        green50: 'rgba(1, 254, 182, 0.4)',
+        'accent-semi-transparent': 'rgba(21, 161, 252, 0.5)',
```

Comment:
Btw, did you know, that you can use `#fed823` + `50` = `#fed82350` to make it half transparent. Maybe we can use hexes here, it will be better

Reply:
If we can't, it's fine

Reply:
Let's leave it for now